conduit-spotify/src/Commands/SpotifyAuthCommand.php

```
@@ -0,0 +1,168 @@
+                    break;
+                case 'Windows':
+                    exec("start '{$url}'");
+                    break;
+                case 'Linux':
+                    exec("xdg-open '{$url}'");
+                    break;
+            }
+        } catch (\Exception $e) {
+            // Silently fail if we can't open browser
+        }
+    }
```

PHPMD (2.15.0)

Avoid unused private methods such as 'openBrowser'. (Unused Code Rules)

(UnusedPrivateMethod)